src/main/java/com/reckue/post/services/realizations/VideoNodeServiceRealization.java

```diff
@@ -0,0 +1,62 @@
+package com.reckue.post.services.realizations;
+
+import com.reckue.post.exceptions.ModelNotFoundException;
+import com.reckue.post.models.VideoNode;
+import com.reckue.post.repositories.VideoNodeRepository;
+import com.reckue.post.services.VideoNodeService;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.util.Comparator;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class VideoNodeServiceRealization implements VideoNodeService {
+
```

Comment:
Please,
- add java docs;
- fix the method update;
- add and fix the method findAll and its derivatives.

For example you can use TextNodeServiceRealization.